ocaml/xapi/message_forwarding.ml

```diff
@@ -3726,13 +3726,16 @@ functor
               ~cert
         )
 
-      let uninstall_ca_certificate ~__context ~host ~name =
-        info "Host.uninstall_ca_certificate: host = '%s'; name = '%s'"
+      let uninstall_ca_certificate ~__context ~host ~name ~force =
```

Comment:
Changing `certificate_uninstall` to pass `~force:false` to `uninstall_ca_certificate` allows the deprecated api function to not be modified in any way.